apps/web/src/app/page.tsx

```diff
@@ -2,22 +2,26 @@
 
 import * as React from "react";
 import { Container, Box, Typography } from "@mui/material";
+import Header from "@/components/header/Header";
 
 export default function Home() {
   return (
-    <Container maxWidth="lg">
-      <Box
-        sx={{
-          my: 4,
-          display: "flex",
-          flexDirection: "column",
-          justifyContent: "center",
-          alignItems: "center",
-        }}>
-        <Typography variant="h4" component="h1" sx={{ mb: 2 }}>
-          Home Page
-        </Typography>
-      </Box>
-    </Container>
+    <>
+      <Header />
```

Comment:
The header component will be shared across other pages based on the UI design, so it should not be limited to only the home page.

Reply:
I think the header will be placed in page.tsx, as the home component is just a placeholder right now. It might eventually be replaced with a component that includes home, settings, and other pages. If I move the header to the root, it will be visible on all pages, including login and welcome pages.

Reply:
I think placing it on the layout component will be better.

Reply:
Fixed it! Also validated the cypress e2e and components test cases!